### DISCUSSION TITLE
Guidance Appreciated | Questions Concerning Lighthouse's Changelog (Version Releases)

### DISCUSSION THREAD

**Discussion by Generosus** [GoogleChrome/lighthouse#14907] [Q&A] [answered]
Good Day,

Can someone kindly assist with the following questions concerning Lighthouse's changelog?

1. Which is the official site (changelog) for checking the version release dates for Lighthouse? Currently, this is the site we're referring to: https://github.com/GoogleChrome/lighthouse/releases

2. From looking at the above site, we noticed the release dates for V10.0.2 and V10.0.1 state, "three weeks ago" and "last month", respectively. In order to determine the actual release dates, we have to hover over those text strings. Is there a reason for this? Why not display the date instead?

3. On average, how soon after a new version Lighthouse is released can we expect to see that version running from within all major browsers that support it?

Thank you :)

**Comment by adamraine** [comment, marked as answer]
> Which is the official site (changelog) for checking the version release dates for Lighthouse? Currently, this is the site we're referring to: [GoogleChrome/lighthouse/releases](https://github.com/GoogleChrome/lighthouse/releases)

You are referencing the correct site for official releases.

> From looking at the above site, we noticed the release dates for V10.0.2 and V10.0.1 state, "three weeks ago" and "last month", respectively. In order to determine the actual release dates, we have to hover over those text strings. Is there a reason for this? Why not display the date instead?

That is just how Github displays the release date. It's not within our control.

> On average, how soon after a new version Lighthouse is released can we expect to see that version running from within all major browsers that support it?

If you are talking about Lighthouse in DevTools, we say what Chrome version it will release in at the top of the release notes. Clicking the version link will take you to Chrome's release schedule which tells you when that Chrome version will be released to stable.

If you are using Lighthouse via the CLI or Node, you can use Lighthouse right away with the latest stable Chrome.

If you are using Pagespeed Insights, it typically takes 2 weeks to ship a new version of Lighthouse but this can vary.

**Reply by Generosus** [reply]
Hey @adamraine,

Excellent reply. Much appreciated. Thank you :)
